holidays/holiday_groups.py

```diff
@@ -662,6 +669,16 @@ def _add_chinese_new_years_day_five(self, name) -> Optional[date]:
             days_delta=+4,
         )
 
+    def _add_ching_ming_festival(self, name) -> date:
```

Comment:
Any objections to use this name instead?
```suggestion
    def _add_qingming_festival(self, name) -> date:
```

Reply:
+1 on this, we should avoid Wade-Giles if possible unless it's better known in the English language that way for standardisation.

Reply:
Renamed. Holiday name in Hong Kong remained "Ching Ming Festival" as in HK Government source.